src/main/java/com/spikes2212/util/AddressableLEDWrapper.java

```diff
@@ -0,0 +1,126 @@
+package com.spikes2212.util;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+
+import java.awt.*;
+
+/**
+ * A class that handles the communication between an LED strip and the code.
+ *
+ * @author Camellia Lami
+ * @see DashboardedSubsystem
+ */
+public class AddressableLEDWrapper {
+
+    /**
+     * {@link AddressableLED} - The LED strip itself.
+     */
+    private final AddressableLED led;
+    /**
+     * {@link AddressableLEDBuffer} - The LED strip's data.
+     */
+    private final AddressableLEDBuffer ledBuffer;
+
+    public AddressableLEDWrapper(int ledPort, int numberOfLEDs) {
+        this.led = new AddressableLED(ledPort);
+        this.ledBuffer = new AddressableLEDBuffer(numberOfLEDs);
+    }
+
+    /**
+     * Sets the LED strip to a specific color.
+     *
+     * @param red   the red value
```

Comment:
specify the range (0 to 1, 0 to 255, something else)
do it for all the other parameters and relevant constrcutors as well